qa/logging-config/src/test/resources/plugin-security.policy

```diff
@@ -0,0 +1,4 @@
+grant {
+  // Needed to read the log file
+  permission java.io.FilePermission "${tests.logfile}", "read";
```

Comment:
It is the integrationTestRunner that needs that. Otherwise we will get:
```
REPRODUCE WITH: ./gradlew :qa:logging-config:integTestRunner -Dtests.seed=6D8D42254DAD541C -Dtests.class=org.elasticsearch.qa.custom_logging.CustomLoggingConfigIT -Dtests.method="testSuccessfulStartupWithCustomConfig" -Dtests.security.manager=true -Dtests.locale=en-IN -Dtests.timezone=Asia/Tel_Aviv -Dtests.distribution=zip -Dcompiler.java=11 -Druntime.java=8
ERROR   1.12s | CustomLoggingConfigIT.testSuccessfulStartupWithCustomConfig <<< FAILURES!
   > Throwable #1: java.security.AccessControlException: access denied ("java.io.FilePermission" "/Users/przemek/workspace/pgomulka/elasticsearch/qa/logging-config/build/cluster/integTestCluster node0/elasticsearch-7.0.0-SNAPSHOT/logs/qa_logging-config_integTestCluster_server.log" "read")
   >    at __randomizedtesting.SeedInfo.seed([6D8D42254DAD541C:F2069750FBE5E1B8]:0)
   >    at java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)

```